lib/fastlane/plugin/wpmreleasetoolkit/helper/ios/ios_git_helper.rb

```diff
@@ -15,14 +15,13 @@ module GitHelper
         # @env PROJECT_NAME The name of the directory containing the project code (especially containing the Resources/ subfolder)
         #
         # @param [Bool] include_deliverfile If true (the default), includes the `fastlane/Deliverfile` in files being commited
-        # @param [Bool] include_metadata If true (the default), includes the `fastlane/download_metadata.swift` file and the `.pot` file (which typically contains an entry or release notes for the new version)
+        # @param [Bool] include_metadata If true (the default), includes the `.pot` file (which typically contains an entry or release notes for the new version)
         #
         def self.commit_version_bump(include_deliverfile: true, include_metadata: true)
-          files_list = [File.join(ENV['PROJECT_ROOT_FOLDER'], 'config', '.')]
+          files_list = [File.join(env_project_root(), 'config', '.')]
           files_list.append File.join('fastlane', 'Deliverfile') if include_deliverfile
           if include_metadata
```

Comment:
I left this branch for the param `include_metadata` as at first I wasn't sure whether it was being used, but now looking at it and considering our recent discussions, I believe this can also go away.